src/devices/CharacterLcd/LcdCharacterEncodingFactory.cs

```diff
@@ -0,0 +1,929 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+
+namespace Iot.Device.CharacterLcd
+{
+    /// <summary>
+    /// Factory for creating Encodings that support different cultures on different LCD Displays. 
+    /// </summary>
+    public class LcdCharacterEncodingFactory
+    {
+        private static Dictionary<char, byte> DefaultA00Map;
```

Comment:
I'm assuming those dictionaries are not being anywhere further except construction? Otherwise this will cause threading issues

Reply:
They're initialized in the static ctor and shouldn't be touched later on. As you say it... I think I need to make sure they're copied upon usage. Good point.